docs/FilterList.md

```diff
@@ -185,3 +185,65 @@ const CustomerList = props => (
 {% endraw %}
 
 **Tip**: The `<FilterList>` Sidebar is not a good UI for small screens. You can choose to hide it on small screens (as in the previous example). A good tradeoff is to use `<FilterList>` on large screens, and the Filter Button/Form combo on Mobile.
+
+## Customize How Filters Are Applied
+
+Sometimes, you may want to customize how filters are applied. For instance, by allowing users to select multiple items such as selecting multiple categories. The `<FilterListItem>` component accepts two props for this purpose:
+
```

Comment:
I'd add a screencast of your story to illustrate this

Reply:
Done